nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/client/ESRestClientFactory.java

```diff
@@ -87,11 +87,11 @@ protected ESClient createLocalRestClient(ElasticSearchEmbeddedServerConfig serve
             throw new IllegalArgumentException(
                     "Embedded configuration has no HTTP port enable, use TransportClient instead of Rest");
         }
-        RestClient lowLevelRestClient = RestClient.builder(
-                new HttpHost("localhost", Integer.parseInt(serverConfig.getHttpPort()))).build();
+        RestClientBuilder lowLevelRestClient = RestClient.builder(
+                new HttpHost("localhost", Integer.parseInt(serverConfig.getHttpPort())));
         RestHighLevelClient client = new RestHighLevelClient(lowLevelRestClient);
         // checkConnection(client);
-        return new ESRestClient(lowLevelRestClient, client);
```

Comment:
Just an observation that this code looks strange.  You used to call a `build()` on a builder but now `build()` is never called.

Reply:
well RestHighLevelClient is now taking a builder instead of client, so build is done in es lib